velox/connectors/hive/storage_adapters/gcs/GcsFileSystem.h

```diff
@@ -77,8 +78,13 @@ class GcsFileSystem : public FileSystem {
   /// google::cloud::storage::Client::ListObjects
   std::vector<std::string> list(std::string_view path) override;
 
-  /// Unsupported
-  void rename(std::string_view, std::string_view, bool) override;
+  /// Renames the original object to new object name by using
+  /// google::cloud::storage::Client::CopyObject and
+  /// google::cloud::storage::Client::DeleteObject
```

Comment:
Let's document that the rename happens by copy and does not guarantee consistency.

Reply:
@majetideepak Updated.